src/lib/consensus/tests/test_proof_of_stake.ml

```diff
@@ -0,0 +1,40 @@
+(** Testing
+    -------
+    Component: Consensus.Proof_of_stake
```

Comment:
This PR also contains moving of Proof_of_stake tests, is this intended?

Reply:
Yes.